schemas/mobility.schema

```diff
@@ -22,4 +22,16 @@ n_trips_250_500 	   INTEGER,
 n_trips_gte_500 	   INTEGER
 );
 
+CREATE TABLE $PG_SCHEMA_MOBILITY.airtravel (
+ts  				   TIMESTAMP NOT NULL,
+origin_code   		   TEXT,
```

Comment:
Quick question. Is this FIPS code or some other code?

Reply:
`origin_code` is the airport code for the origin city, for example `PIT` for Pittsburgh International Airport

Reply:
Right.